src/controllers/AuthController.js

```diff
@@ -0,0 +1,43 @@
+import jwt from 'jsonwebtoken';
+import helpers from '../helpers';
```

Comment:
Nice work!
I feel the name helpers can be refactored to be more specific because there will be other files in the helper folder and naming the variable helper can confuse others

Reply:
The reason for the index.js file in the helpers folder is so that all helpers can be imported in one place. Imagine if 10 helpers are needed in a controller class and you want to import them directly, that would mean 10 import statements which isn't nice. With the index.js file, you can just `import helpers from '../helpers'` and destructure the particular helper(s) you need from it. This is much cleaner than importing each helper directly.

Reply:
LGTM